lib/wallaby/experimental/chrome/chromedriver.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Wallaby.Experimental.Chrome.Chromedriver do
+  use GenServer
+
+  @external_resource "priv/run_phantom.sh"
+  @script_contents File.read! "priv/run_phantom.sh"
+
+  def start_link(_args) do
+    GenServer.start_link(__MODULE__, [])
+  end
+
+  def stop(server) do
+    GenServer.stop(server)
+  end
+
+  def base_url(server) do
+    GenServer.call(server, :base_url)
+  end
+
+  def init(_) do
+    tcp_port = find_available_port()
+    port = start_chromedriver(tcp_port)
+
+    {:ok, %{running: false, port: port, base_url: "http://localhost:#{tcp_port}/"}}
+  end
+
+  def handle_call(:base_url, _from, %{base_url: base_url}=state) do
+    {:reply, {:ok, base_url}, state}
+  end
+
+  def terminate(reason, state) do
+    IO.puts("terminating")
+  end
+
+  defp find_available_port() do
+    {:ok, listen} = :gen_tcp.listen(0, [])
+    {:ok, port} = :inet.port(listen)
+    :gen_tcp.close(listen)
+    port
+  end
```

Comment:
I thinkw e have this function already in Phantom? Might be worth extracting and reusing? :)